worlds/paint/__init__.py

```diff
@@ -0,0 +1,81 @@
+from typing import Dict, Any
+
+from BaseClasses import Tutorial, Region, Entrance
+from worlds.AutoWorld import World, WebWorld
+from .items import item_table, PaintItem, item_data_table
+from .locations import location_table, PaintLocation, location_data_table
+from .options import PaintOptions
+
+
+class PaintWebWorld(WebWorld):
+    theme = "partyTime"
+
+    setup_en = Tutorial(
+        tutorial_name="Start Guide",
+        description="A guide to playing Paint in Archipelago.",
+        language="English",
+        file_name="guide_en.md",
+        link="guide/en",
+        authors=["MarioManTAW"]
+    )
+
+    tutorials = [setup_en]
+
+
+class PaintWorld(World):
```

Comment:
Missing the docstring on the World class, which is how the webhost populates the game description on the "Supported Games" page